examples/starwars_relay/schema.py

```diff
@@ -1,34 +1,27 @@
-import graphene
+from graphene import ObjectType, Schema, Field, String
 from graphene import relay
```

Comment:
Why keep this as a module import rather than importing `relay` classes from `graphene` as well (all are available in the top level module) or import the classes from `graphene.relay`?

Reply:
I wanted to preserve this as a reminder to revisit this in conversation. Do we want to go completely explicit, or just top-level to highlight the sub-package. I'm waffling back and forth in my head on this.